### PR TITLE
Update v17.12.3

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

File: org.kde.kdenlive.json [hunk 1]
```diff
@@ -431,13 +431,13 @@
...
-                    "url": "https://download.kde.org/stable/applications/17.12.2/src/kdenlive-17.12.2.tar.xz",
-                    "sha256": "e07064d2c2041ae3d59bf715350e911a863abf8ea46a8388f2993bced1d95ad0"
+                    "url": "https://download.kde.org/stable/applications/17.12.3/src/kdenlive-17.12.3.tar.xz",
+                    "sha256": "371f860365d01c4c0aebe931354cc99a646d44f17e6846c93ee2c002258f503e"
                 },
                 {
                     "type": "shell",
                     "commands": [
-                        "sed -i 's|<releases>|<releases><release date=\"2018-02-06\" version=\"17.12.2\"/>|' data/org.kde.kdenlive.appdata.xml"
+                        "sed -i 's|<releases>|<releases><release date=\"2018-02-06\" version=\"17.12.3\"/>|' data/org.kde.kdenlive.appdata.xml"
```
Comment (TingPing) [inline org.kde.kdenlive.json]: Date too.

**Comment by aleixpol** [comment]
I don't really understand why this is needed. Shouldn't this be in the repository upstream?

**Comment by TingPing** [comment]
Yes.

**Comment by eszlari** [comment]
see https://github.com/flathub/org.kde.kdenlive/commit/21d120d78758ac03135e61e3daa80e6b193d0567

**Pull request closed without merging**